assets/training/finetune_acft_hf_nlp/components/model_converter/common/spec.yaml

```diff
@@ -0,0 +1,32 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: ft_nlp_model_converter
+version: 0.0.21
+type: command
+
+is_deterministic: True
+
+display_name: Common Model Converter
+description: Component to convert the finetune job output to pytorch and mlflow model
+
+environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/25
+
+code: ../../../src/model_converter
```

Comment:
Can we refer code from src like following and update the command as well:

`code: ../../../src`

This will allow component to access all files inside src like utils and others

Reply:
Will do it in the upcoming PR's